assets/helpers/abtest.js

```diff
@@ -157,46 +125,59 @@ function assignUserToVariant(mvtId: number, test: Test): string {
   return test.variants[variantIndex];
 }
 
-function getParticipation(abTests: Test[], mvtId: number, country: IsoCountry): Participations {
+function getParticipations(abTests: Tests, mvtId: number, country: IsoCountry): Participations {
 
   const currentParticipation = getLocalStorageParticipation();
-  const participation:Participations = {};
+  const participations: Participations = {};
 
-  abTests.forEach((test) => {
+  Object.keys(abTests).forEach((testId) => {
+    const test = abTests[testId];
 
     if (!test.isActive) {
       return;
     }
 
-    if (test.testId in currentParticipation) {
-      participation[test.testId] = currentParticipation[test.testId];
+    if (testId in currentParticipation) {
+      participations[testId] = currentParticipation[testId];
     } else if (userInTest(test.audiences, mvtId, country)) {
-      participation[test.testId] = assignUserToVariant(mvtId, test);
+      participations[testId] = assignUserToVariant(mvtId, test);
     } else {
-      participation[test.testId] = 'notintest';
+      participations[testId] = 'notintest';
     }
-
   });
 
-  return participation;
-
+  return participations;
 }
 
+const buildOphanPayload = (participations: Participations, complete: boolean): OphanABPayload =>
+  Object.keys(participations).reduce((payload, participation) => {
+    const ophanABEvent: OphanABEvent = {
+      variantName: participations[participation],
+      complete,
+      campaignCodes: [],
+    };
 
-// ----- Exports ----- //
+    return Object.assign({}, payload, { [participation]: ophanABEvent });
+  }, {});
 
-const init = (country: IsoCountry, abTests: Test[] = tests): Participations => {
+// ----- Exports ----- //
 
-  const mvt: number = getMvtId();
-  let participation: Participations = getParticipation(abTests, mvt, country);
+const trackAB = (participations: Participations, complete: boolean): void => {
```

Comment:
Since we have 3 type of tracking (GA, third party and acquisition), is it worth to change this name to `trackABOphan`?